databricks_asset_bundles/04.deploy_model.py

```
@@ -0,0 +1,54 @@
+workspace.serving_endpoints.update_config_and_wait(
+    name="nyctaxi-model-serving-fe",
+    served_entities=[
+        ServedEntityInput(
+            entity_name=f"{catalog_name}.{schema_name}.nyctaxi_model_fe",
+            scale_to_zero_enabled=True,
+            workload_size="Small",
+            entity_version=model_version,
+        )
+    ],
+)
+print("done")
```

Ruff (0.7.0)

No newline at end of file

Add trailing newline

(W292)